agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -4507,6 +4506,31 @@ bool backhaul_manager::handle_backhaul_steering_request(ieee1905_1::CmduMessageR
     return send_cmdu_to_bus(cmdu_tx, controller_bridge_mac, bridge_info.mac);
 }
 
+bool backhaul_manager::create_backhaul_steering_response(
+    const sMacAddr &sta_mac, const sMacAddr &target_bssid,
+    const wfa_map::tlvBackhaulSteeringResponse::eResultCode &result_code)
+{
+    auto cmdu_tx_header = m_expected_backhaul_sta_steering.response.create(
+        0, ieee1905_1::eMessageType::BACKHAUL_STEERING_RESPONSE_MESSAGE);
```

Comment:
Since we are not building up the message from different responses here but we're just expecting a single one, there's no need to keep a response buffer in m_expected_backhaul_sta_steering. We can simply use cmdu_tx here.
```suggestion
    auto cmdu_tx_header = cmdu_tx.create(
        0, ieee1905_1::eMessageType::BACKHAUL_STEERING_RESPONSE_MESSAGE);
```

Reply:
accepted